practice/Capstone/test_commandutility.py

```diff
@@ -0,0 +1,133 @@
+import pytest
+from main import *
+import tempfile
+from unittest.mock import Mock
+import shutil
+import re
+
+config = configparser.ConfigParser()
+config.read('test.ini')
+correct_schemas = [config[section_name]['data_schema'] for section_name in config.sections()
+                   if 'correct' in section_name]
+correct_paths = ['.', './', '/Users/imacedonski/PycharmProjects/PYTHON-BASIC/practice/Capstone/output1', './output']
```

Comment:
your local path will not work in other environments, replace with the parametrized one

Reply:
Done.